### ISSUE TITLE
cluster-autoscaler: Unable to look up PVC info 

### ISSUE THREAD

**Issue by yawboateng** [kubernetes/contrib#2507]
cluster-autoscaler works fine but im seeing multiple lines of this error in the logs:
`
E0328 18:14:57.140465       1 predicates.go:234] Unable to look up PVC info for logging/datadir-elasticsearch-data-ss-1, assuming PVC matches predicate when counting limits: persistentvolumeclaim "datadir-elasticsearch-data-ss-1" not found `

I've confirmed the pvc datadir-elasticsearch-data-ss-1does exist in the logging namespace

**Comment by olvesh** [comment]
I also see the same, and when the autoscaler should have executed a scale up it didn't happen. 
The pod that should have caused the scaling was also using a PVC, and the event on the pod listed:
```  
FirstSeen	LastSeen	Count	From			SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----			-------------	--------	------			-------
  27m		17m		55	{cluster-autoscaler }			Normal		NotTriggerScaleUp	pod didn't trigger scale-up (it wouldn't fit if a new node is added)
  17m		17m		1	{cluster-autoscaler }			Normal		NotTriggerScaleUp	pod didn't trigger scale-up (it wouldn't fit if a new node is added)
  16m		16m		1	{cluster-autoscaler }			Normal		TriggeredScaleUp	pod triggered scale-up, group: nodes.example.com, sizes (current/new): 2/3
```

Last line came after I swapped the 0.5.0 for the 0.4.0 of cluster-autoscaler.

Using Kops - if that has something to do with this.

**Comment by mwielgus** [comment]
Which version of K8S are you on?

**Comment by olvesh** [comment]
1.5.4

**Comment by yawboateng** [comment]
and im on v1.5.3

**Comment by MaciekPytel** [comment]
This has been fixed in cluster-autoscaler 0.5.2. Since CA image has now been released I'm closing the issue.
